ansible/server-state-playbooks/ome-dundeeomero/playbook.yml

```diff
@@ -11,6 +11,27 @@
            ((ansible_virtualization_type is defined)
            and (ansible_virtualization_type == "VMware"))
 
+    # Not required for an OMERO server install - UoD specific implementation
```

Comment:
Agreed - I definitely do see the value of these reviews - I didn't skip review because I thought otherwise - rather to progress `ome-demoserver` and have all my ansible code in one (new) branch such that I could diff / merge from my previous code.